source/_static/scss/includes/_variables.scss

```diff
@@ -2,8 +2,20 @@
 $white: #ffffff;
 $black: #000000;
 $theme-red: #C72C48;
+$theme-red-light: rgba($theme-red, 0.5);
+$theme-red-lighter: rgba($theme-red, 0.25);
 $theme-hover-bg: rgb(245, 245, 245);
 
+// Overrides for sphinx-design
+// https://sphinx-design.readthedocs.io/en/alabaster-theme/css_variables.html
+
+:root {
```

Comment:
@rushenn @kaankabalak 

overrides for sphinx-design components. This seemed to work, but I'm not sure if using `:root` is the correct thing to do here.

Reply:
Adding CSS variables isnide `:root`, make them accessible from anywhere in the stylesheet. I think this is fine as we don't have any specific style config requirements.